### PR TITLE
Cross-browser compatible styling for input[type=file]

### LINKED ISSUE #5423: Cross-Browser Compatibility With `input type=file`
The issue is, Firefox ignores the line-height property set for `input type=file` while Chrome doesn't. The resulting difference can be seen here:

[https://publiclab.org/post](https://publiclab.org/post) 

here is my view on MacOS 10.14.3 and Chrome: - pretty off
Screen Shot 2019-04-09 at 1 44 10 AM

here is my view on same OS and Firefox: - perfect
Screen Shot 2019-04-09 at 1 45 08 AM

same OS and Safari: - not perfect but fine
Screen Shot 2019-04-09 at 1 52 18 AM

**solution**

- Set the `line-height: 0` for all `input type=file`

will be opening a PR for this shortly to show results

### PR DESCRIPTION
**Pull request by sashadev-sky**
Fixes #5423  (<=== Add issue number here)

===============================================
After applying a `line-height: 0` globally to `input[type=file]`:

Chrome: improved - now better
chome-fix

Firefox: stays the same (perfect)
firefox-fix

Safari: improved -now perfect
safari-fix

===============================================

OLD ones pasted in from original issue for easy reference:
before after

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@publiclab/reviewers 
also any reviewers that are not on mac would be great because that is what I'm using!! and please say your OS when you review

**Comment by sashadev-sky** [comment]
@divyabaid16 @shivam15 can I have review here too please :)

**Comment by shivam15** [comment]
@sashadev-sky  Works for me!!

**Comment by sashadev-sky** [comment]
@gauravano can I please get a 2nd review on this so we can merge :)

**Comment by divyabaid16** [comment]
It works for me too @sashadev-sky
:smile:

**Comment by sashadev-sky** [comment]
@shivam15 @divyabaid16 Thank you to both of you :) I really appreciate you taking the time! 

@jywarren this one is ready for merge

**Comment by jywarren** [comment]
Awesome, thank you!!!

File: app/assets/stylesheets/cross_browser.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* 
+================================================================
```
Comment (gauravano) [inline app/assets/stylesheets/cross_browser.css]: Really awesome!

**Pull request merged**